bin/reth/src/node/mod.rs

```diff
@@ -141,10 +141,16 @@ impl Command {
         raise_fd_limit();
 
         let mut config: Config = self.load_config()?;
-        info!(target: "reth::cli", path = %self.db, "Configuration loaded");
 
-        info!(target: "reth::cli", path = %self.db, "Opening database");
-        let db = Arc::new(init_db(&self.db)?);
+        // add network name to db directory
+
+        let db_path = self.db.join(self.chain.chain.to_string());
```

Comment:
I would prefer if we used a chain ID here to make sure minor changes in naming etc. does not break the DB.